ash/src/device.rs

```diff
@@ -2008,6 +2008,29 @@ pub trait DeviceV1_0 {
             .get_render_area_granularity(self.handle(), render_pass, &mut granularity);
         granularity
     }
+
+    #[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkGetDeviceMemoryCommitment.html>"]
+    unsafe fn get_device_memory_commitment(
+        &self,
+        memory: vk::DeviceMemory,
+    ) -> vk::DeviceSize {
+        let mut committed_memory_in_bytes = mem::zeroed();
+        self.fp_v1_0()
+            .get_device_memory_commitment(self.handle(), memory, &mut committed_memory_in_bytes);
+        committed_memory_in_bytes
+    }
+
+    #[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkGetImageSparseMemoryRequirements.html>"]
+    unsafe fn get_image_sparse_memory_requirements(&self, image: vk::Image) -> Vec<vk::SparseImageMemoryRequirements> {
+        let mut num = 0;
```

Comment:
should be sparse_memory_requirement_count ?

Reply:
Potentially. I tried to keep consistent with existing function bodies, for example `enumerate_device_extension_properties` and `enumerate_physical_devices`.

Reply:
`get_image_sparse_memory_requirements2` uses `count` as well so there's that. Since this is based on the Vulkan headers where `pSparseMemoryRequirementCount` is used, let's go for `count` as the rest of the name is already implied by the function name.

Reply:
By the same rules it looks like the other `num` variables below should be renamed to `count` too.